### PR TITLE
create react components-module-1

### PR DESCRIPTION
**Pull request by johari02shubhi**
Created components,

Counter
A component that renders a numeric value and two buttons: one to decrement the value by 1, another to increment the value by 1.
The component should take an initial value in a property.
The component should be written using EcmaScript classes syntax, the render method should use React.createElement API (without JSX).

SearchForm
The component should accept two properties:
Initial search query. Use the value to set the initial value for the input
A "onSearch" callback property. Call the callback property every time the user presses Enter when the input has focus or when the user clicks the Search button. Pass current input value in callback arguments.

GenreSelect
A list of genre names to display. Use the incoming list to render genre buttons.
A name of currently selected genre. Use the name to identify which button to highlight.
A "onSelect" callback property. Call the callback function when the user clicks on any genre button. Pass respective genre name to the callback arguments.

Visuals:
image

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Counter from './components/counter.js';
+import SearchForm from './components/searchform.js';
+import GenreSelect from './components/genreselect.js';
```
Comment (AB619) [inline src/index.js]: Please correct these paths.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    return (
+      <div className="div-container">
+        <Counter initialValue={0} />
+        <SearchForm
+          initialQuery="What do you want to watch?"
+          onSearch={this.handleSearch}
+        />
+        <GenreSelect
+          genres={['All', 'Documentary', 'Comedy', 'Horror', 'Crime']}
+          selectedGenre={selectedgenre}
```
Comment (AB619) [inline src/index.js:37]: Initial genre is not being displayed in UI. Please look into it.

Comment (johari02shubhi) [reply src/index.js:37]: Done! commit id: https://github.com/johari02shubhi/netflixroulette/pull/1/commits/766d0bfb6414ddd6cb648f9e35b484a9d58fb32f

image

File: src/index.js (hunk 1 above)
Comment (johari02shubhi) [reply src/index.js]: Done! commit id: - https://github.com/johari02shubhi/netflixroulette/pull/1/commits/766d0bfb6414ddd6cb648f9e35b484a9d58fb32f

Comment (AB619) [reply src/index.js]: Correct path still missing.
image

Comment (AB619) [reply src/index.js]: Correction: 
`import Counter from './components/Counter';
import GenreSelect from './components/GenreSelect';`

File: src/index.js (hunk 2 above)
Comment (AB619) [reply src/index.js:37]: lgtm

File: src/index.js (hunk 1 above)
Comment (johari02shubhi) [reply src/index.js]: @AB619 updated with commit id: https://github.com/johari02shubhi/netflixroulette/pull/1/commits/b191cfbf7969881de2dce2c2c9da8099ee938bcf

**Review by AB619 — COMMENTED** [review]
lgtm

**Pull request merged**